flow/activities/flowable_core.go

```diff
@@ -453,6 +453,7 @@ func replicateQRepPartition[TRead any, TWrite StreamCloser, TSync connectors.QRe
 			return a.Alerter.LogFlowError(ctx, config.FlowJobName, fmt.Errorf("[qrep] failed to pull records: %w", err))
 		}
 
+		// should only be effective for Postgres source, others report within connector
```

Comment:
this is still used by mysql since it returns whatever wasn't reported by goroutine yet at end